src/content/docs/pt-br/reference/configuration-reference.mdx

```diff
@@ -503,60 +508,81 @@ de configuração especiais para redirecionamentos e não precisam/querem redire
 </p>
 
 Controla se os estilos do projeto são enviados para o navegador em um arquivo CSS separado ou em linha em tags `<style>`. Escolha entre as seguintes opções:
- - `'always'` - estilos do projeto são embutidos em tags `<style>`
- - `'auto'` - apenas folhas de estilo menores que `ViteConfig.build.assetsInlineLimit` (padrão: 4kb) são embutidas. Caso contrário, estilos do projeto são enviados em folhas de estilo externas.
- - `'never'` - estilos do projeto são enviados em folhas de estilo externas
+  - `'always'` - estilos do projeto são embutidos em tags `<style>`
+  - `'auto'`   - apenas folhas de estilo menores que `ViteConfig.build.assetsInlineLimit` (padrão: 4kb) são embutidas. Caso contrário, estilos do projeto são enviados em folhas de estilo externas.
+  - `'never'`  - estilos do projeto são enviados em folhas de estilo externas
 
 ```js
 {
-	build: {
-		inlineStylesheets: `never`,
-	},
+  build: {
+    inlineStylesheets: `never`,
+  },
 }
 ```
 
-
-### build.split
+## Opções de pré-carregamento
 
 <p>
 
-**Tipo:** `boolean`<br />
-**Padrão:** `false`
+**Tipo:** `boolean | object`
 </p>
 
-A opção de configuração da build `build.split` foi substituída pela opção de configuração do adaptador [`functionPerRoute`](/pt-br/reference/adapter-reference/#functionperroute).
+Habilita o pré-carregamento de links no seu site para oferecer transições de páginas mais rápidas.
+(Habilitado por padrão em páginas que usam o roteador `<ViewTransitions />`. Defina `prefetch: false` para optar por não ter esse comportamento.)
 
-Por favor veja a [documentação do seu adaptador SSR](/pt-br/guides/integrations-guide/#integrações-oficiais) para utilizar `functionPerRoute` para definir como seu código SSR deve passar por bundle.
+Essa configuração adiciona um script de pré-carregamento automaticamente em todas as páginas no projeto
+lhe dando acesso ao atributo `data-astro-prefetch`.
+Adicione esse atributo em qualquer link `<a />` em sua página para habilitar o pré-carregamento para aquela página.
 
-```js
-{
-  build: {
-    split: true
-  }
-}
+```html
+<a href="/sobre" data-astro-prefetch>Sobre</a>
 ```
+Customize mais o comportamente padrão de pré-carregamento utilizando as opções [`prefetch.defaultStrategy`](#prefetchdefaultstrategy) e [`prefetch.prefetchAll`](#prefetchprefetchall).
```

Comment:
```suggestion
Customize ainda mais o comportamente padrão de pré-carregamento utilizando as opções [`prefetch.defaultStrategy`](#prefetchdefaultstrategy) e [`prefetch.prefetchAll`](#prefetchprefetchall).
```
Suggestion: I think `Customize ainda mais` would be a more accurate and friendly translation to `Further customize`.

Reply:
The text up till this point is not talking about any customization, so this seems weird to emphasize adding _more_ customization.

Seems tricky to me because the original "Further customize" means to add customization on top of what was discussed, it doesn't imply that there were customization before.

Or maybe that is just me